.dev/sass/layouts/_header.scss

```diff
@@ -68,7 +68,7 @@ body.no-max-width .site-header-wrapper {
 		position: relative;
 		clear: both;
 
-		h1 {
+		.page-title {
```

Comment:
Yeah, you are right. Let me take a second peak at this and see if there is another workaround.